### PR TITLE
Adds Syntax Highlighting for E-Mail files (*.eml)

### PR DESCRIPTION
**Pull request by mariozaizar**
https://github.com/mariozaizar/eml-tmLanguage

demo

### CODE REVIEW

**Comment by mariozaizar** [comment]
*Updated*
- Created tag in repository.
- Moved away of branch-based release.

**Comment by wbond** [comment]
You should remove the files `.no-sublime-package` and `package-control.cleanup` from your repo.

File: repository/e.json [hunk 1]
```diff
@@ -503,6 +503,20 @@
 			]
 		},
 		{
+			"name": "EML (E-Mail)",
+			"labels": ["eml", "email", "mail", "mime", "e-mail", "RFC2045", "RFC2046", "RFC2047", "RFC4288", "RFC4289", "RFC2049"],
+			"details": "https://github.com/mariozaizar/eml-tmLanguage",
+			"issues": "https://github.com/mariozaizar/eml-tmLanguage/issues",
```
Comment (wbond) [inline repository/e.json]: This is the default issues URL based on `details`, so you may omit it.

**Comment by wbond** [comment]
Once the files are deleted it should be ready for merging.

**Comment by mariozaizar** [comment]
@wbond Thanks for feedback!, the [files got deleted](https://github.com/mariozaizar/eml-tmLanguage/commit/8078727d9ac00fad92736219ffa596de5cc62396), and issues url removed.

**Pull request merged**